deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/ProcessorUtils.kt

```diff
@@ -50,3 +53,19 @@ fun XTypeElement.directlyImplementsInterfaces(fqnList: List<String>): Boolean {
         getSuperInterfaceElements().any { typeElement -> typeElement.qualifiedName == interfaceFqn }
     }
 }
+
+inline fun <reified T> XAnnotation.getAsList(method: String): List<T> {
+    val originalList = get<List<T>>(method)
+    // In new XProcessing versions List values are wrapped in XAnnotationValue but in old versions
```

Comment:
Maybe worth putting a todo in the xprocessing version that when it is bumped that this can be simplified.